src/pages/VenuePage/TemplateWrapper.tsx

```diff
@@ -103,7 +104,25 @@ export const TemplateWrapper: React.FC<TemplateWrapperProps> = ({ venue }) => {
         );
       }
       break;
-
+    case VenueTemplate.externalroom:
+      if (venue.zoomUrl) {
+        template = <ExternalRoom zoomUrl={venue.zoomUrl} />;
+      } else {
+        template = (
+          <p>
+            Venue {venue.name} should redirect to a URL, but none was set.
+            <br />
+            <button
+              role="link"
+              className="btn btn-primary"
+              onClick={() => history.goBack()}
+            >
+              Go Back
+            </button>
+          </p>
+        );
+      }
```

Comment:
this should be rendered inside the room

Reply:
fixed in 03fcfc07bda6c3d8df4d15a25ea8f36bb1956118